nsv13/code/modules/munitions/ammunition/torpedos/torpedo_types.dm

```diff
@@ -16,6 +16,13 @@
 	explode_when_hit = TRUE //Yeah, this can't ever end well for you.
 	var/claimable_gulag_points = 75
 
+/obj/item/ship_weapon/ammunition/torpedo/Initialize()
```

Comment:
Irrelevant to ship PR, please move to separate PR.

Reply:
Separated.